### PR TITLE
ant lions

### PR DESCRIPTION
**Pull request by freerangeandy**
can drop deadly ant lions onto field (toggle food/lion in bottom controls)

### CODE REVIEW

File: AntFarm.Ant.js [hunk 1]
```diff
@@ -149,14 +149,16 @@ AntFarm.Ant = Class.extend({
...
-    ant.die = function() {
+    ant.die = function(val) {
       ant.remove();
 
       field.canvas.save();
         // to do rotation, we need a better origin
         // field.canvas.rotate(ant.direction / 180 * Math.PI);
-        field.canvas.fillStyle = "green";
-        field.canvas.fillRect(ant.x - ant.width/2, ant.y - ant.height/2, ant.width, ant.height);
+        if (typeof val == "undefined") {
```
Comment (jywarren) [inline AntFarm.Ant.js:158]: Is this a bugfix? Lol 🐜

**Review by jywarren — APPROVED** [review]
Nice. Seems backwards compatible too?

File: AntFarm.Ant.js (hunk 1 above)
Comment (freerangeandy) [reply AntFarm.Ant.js:158]: when an ant dies by lion, I wanted the ability to turn off the green corpse

**Comment by freerangeandy** [comment]
The change to double-click behavior in Field might need more work before it's fully backwards compatible

**Comment by freerangeandy** [comment]
fixed Field

**Comment by freerangeandy** [comment]
do i press Close and comment now?

**Comment by jywarren** [comment]
No, I have to merge it. If you think it's complete I'll do so?

**Comment by freerangeandy** [comment]
I've finished my changes, so it's good to go.

**Comment by jywarren** [comment]
Awesome, thanks! 🐜 🔥

**Comment by jywarren** [comment]
Live! http://ant.farmfarm.org/examples/antlions

**Comment by jywarren** [comment]
Nice! 

image

Should we put a link into the README?

**Pull request merged**